packages/framework/src/Component/Domain/Multidomain/MultiDomainEntityClassProviderInterface.php

```diff
@@ -2,15 +2,28 @@
 
 namespace Shopsys\FrameworkBundle\Component\Domain\Multidomain;
 
+/**
+ * Interface MultiDomainEntityClassProviderInterface
+ *
+ * Serves as provider for ignored and manual entities.
+ *
+ * MultidomainEntityClassFinder finds entities that has identifier called $domainId, but sometimes this assumption
```

Comment:
Actually, it finds entities that have composite identifier including field called `$domainId`. I would also omit the "...some reason" part. So the whole annotation could look like this:
"MultidomainEntityClassFinder finds multidomain entities, i.e. entities that have composite identifier including field called $domainId. This interface adds option to ignore some of these or manually add entities that do not match the condition when multidomain data are created. See Shopsys\FrameworkBundle\Command\CreateDomainsDataCommand"

Reply:
Do we really want to reference FrameworkBundle? Personally I think not, i would omit the last sentence.